roles/confluent.kerberos/templates/krb5.conf.j2

```diff
@@ -0,0 +1,30 @@
+[logging]
+ default = FILE:/var/log/krb5libs.log
+ kdc = FILE:/var/log/krb5kdc.log
+ admin_server = FILE:/var/log/kadmind.log
+
+[libdefaults]
+ default_realm = {{ realm_name|upper() }}
+ dns_lookup_realm = {{ dns_lookup_realm }}
+ dns_lookup_kdc = {{ dns_lookup_kdc }}
+ ticket_lifetime = {{ ticket_lifetime }}
+ renew_lifetime = {{ renew_lifetime }}
+ forwardable = {{ forwardable }}
+ udp_preference_limit = {{ udp_preference_limit }}
+ default_tkt_enctypes = {{ default_tkt_enctypes }}
+ default_tgs_enctypes = {{ default_tgs_enctypes }}
+ permitted_enctypes = {{ permitted_enctypes }}
+
+[realms]
+ {{ realm_name| upper() }} = {
```

Comment:
realm not realm_name
realm is whats documented to be in hosts file, realm_name is set to confluent.example.com which is wrong for customers

Reply:
created https://confluentinc.atlassian.net/browse/PLAT-601